microservice/pom.xml

```diff
@@ -207,6 +208,12 @@
                 <version>1.8.13</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.hibernate</groupId>
+                <artifactId>hibernate-validator</artifactId>
+                <version>6.1.5.Final</version>
```

Comment:
So I recently removed this dependency, hibernate validator was moved to a new package `org.hibernate.validator`: https://mvnrepository.com/artifact/org.hibernate.validator/hibernate-validator/6.2.3.Final and we should use version defined by spring-boot: https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.9/spring-boot-dependencies-2.5.9.pom

Reply:
Same here, as part of conflict resolution it seems I have reintroduced this dependency again. It is not needed by my code. I removed it again, code works without it.